sklearn/ensemble/tests/test_stacking.py

```diff
@@ -561,3 +566,58 @@ def fit(self, X, y):
     msg = "'MyEstimator' object has no attribute 'n_features_in_'"
     with pytest.raises(AttributeError, match=msg):
         stacker.n_features_in_
+
+
+@pytest.mark.parametrize("stack_method", ["auto", "predict"])
+@pytest.mark.parametrize("passthrough", [False, True])
+def test_stacking_classifier_multilabel(stack_method, passthrough):
+    X_train, X_test, y_train, y_test = train_test_split(
+        X_multilabel_r, y_multilabel_r, stratify=y_multilabel_r, random_state=42
+    )
+
+    estimators = [
+        ("knnc", KNeighborsClassifier()),
+        ("dcs", DummyClassifier(strategy="stratified", random_state=42)),
+        ("dcu", DummyClassifier(strategy="uniform", random_state=42)),
+    ]
+    final_estimator = KNeighborsClassifier()
+
+    clf = StackingClassifier(
+        estimators=estimators,
+        final_estimator=final_estimator,
+        passthrough=passthrough,
+        stack_method=stack_method,
+    )
+    clf.fit(X_train, y_train)
+    clf.predict(X_test)
+    clf.predict_proba(X_test)
+    sc = clf.score(X_test, y_test)
+    # (passthrough & stack_method == "predict") => (sc == 0.44)
+    assert not passthrough or stack_method != "predict" or sc == pytest.approx(0.44)
+
+    # (passthrough & stack_method != "predict") => (sc == 0.6)
+    assert not passthrough or stack_method == "predict" or sc == pytest.approx(0.6)
+
+    # (not passthrough & stack_method == "predict") => (sc == 0.2)
+    assert passthrough or stack_method != "predict" or sc == pytest.approx(0.2)
+
+    # (not passthrough & stack_method != "predict") => (sc == 0.4)
+    assert passthrough or stack_method == "predict" or sc == pytest.approx(0.4)
+
+    X_trans = clf.transform(X_test)
+    expected_column = X_trans.shape[1]
+
+    # (passthrough & stack_method == "predict") => (columns == 29)
+    assert not passthrough or stack_method != "predict" or expected_column == 29
+
+    # (passthrough & stack_method != "predict") => (sc == 38)
+    assert not passthrough or stack_method == "predict" or expected_column == 38
+
+    # (not passthrough & stack_method == "predict") => (sc == 9)
+    assert passthrough or stack_method != "predict" or expected_column == 9
+
+    # (not passthrough & stack_method != "predict") => (sc == 18)
+    assert passthrough or stack_method == "predict" or expected_column == 18
+
+    if passthrough:
+        assert_allclose(X_test, X_trans[:, -20:])
```

Comment:
@jmloyola 
Do you think the tests errors are happening from this code here?  @nestornav and I are trying to figure out where and why the tests are failing.

Reply:
Yes. When I run the test locally the line `assert passthrough or stack_method != "predict" or sc == pytest.approx(0.2)` fails with:

>    assert passthrough or stack_method != "predict" or sc == pytest.approx(0.2)
    E       AssertionError: assert (False or 'predict' != 'predict' or 0.12 == 0.2 ± 2.0e-07)
    E        +  where 0.2 ± 2.0e-07 = <function approx at 0x7f26c3af9430>(0.2)
    E        +    where <function approx at 0x7f26c3af9430> = pytest.approx

Reply:
If I comment that line, the test pass correctly.

Reply:
@jmloyola Tests were written by my partner. I'm not sure why he choose them. I tried to understand but I'm not pretty sure why. 

Taking into account the opened time of this PR, what do you think about skip just the wrong one?